java-checks-test-sources/src/main/java/checks/spring/ControllerWithRestControllerReplacementCheck.java

```diff
@@ -0,0 +1,79 @@
+package checks.spring;
+
+import javax.annotation.Nullable;
+import org.springframework.stereotype.Controller;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+public class ControllerWithRestControllerReplacementCheck {
+
+  @Controller // Noncompliant [[sc=3;ec=14;secondary=14;quickfixes=qf1]] {{Replace the "@Controller" annotation by "@RestController" and remove all
```

Comment:
We are able to offer the removal of the `@ResponseBody` annotations as part of the quickfix, but to complete the operation we should also replace the `@Controller` annotation, shouldn't we?

Reply:
Yes! It is done.